client/client.go

```diff
@@ -19,11 +25,80 @@ func (c *Client) Logger() *zerolog.Logger {
 	return &c.logger
 }
 
+func (c *Client) AppID() int64 {
+	return c.appID
+}
+
+func (c *Client) InstallationID() int64 {
+	return c.installationID
+}
+
+func (c *Client) PrivateKey() string {
```

Comment:
Why are we using getters instead of making the credential fields public?

Reply:
The getter approach maintains security and encapsulation while providing necessary access to the credential data. This is especially important for authentication credentials that should be handled carefully.